atomic_reactor/utils/resource_manager.py

```diff
@@ -273,14 +275,20 @@ def from_config(cls, config: dict):
           enabled: true
           auth: qa-vm-secret-filepath
           username: cloud-user
+          slots: 3
           ...
         hostname-remote-host2:
           ...
         """
-
-        # Add enabled clusters to list hosts
-        hosts = [RemoteHost(hostname=hostname, username=attr['username'], ssh_keyfile=attr['auth'])
-                 for hostname, attr in config.items() if attr['enabled']]
+        hosts = []
+        for hostname, attr in config.items():
+            if not attr.get("enabled", False):
+                continue
+            host = RemoteHost(
+                hostname=hostname, username=attr["username"], ssh_keyfile=attr["auth"]
+            )
+            host.slots = attr.get("slots", 1)
```

Comment:
Could `slots` be set only in `__init__`, like all the other attributes?

Reply:
Easy to change but I'm thinking on having a new `Slot` class and only expose necessary APIs after we make a deal with solution to the locking issues, will probably rework this part of code.

Reply:
Ack :+1: